resources/lib/ipwww_video.py

```diff
@@ -1049,10 +1070,36 @@ def RemoveWatching(episode_id):
 
 
 def ListFavourites():
-    url = "https://www.bbc.co.uk/iplayer/added"
-    data = GetJsonDataWithBBCid(url)
-    if data:
-        ParseJSON(data, url)
+    """AKA Added"""
+    data = GetJsonDataWithBBCid("https://www.bbc.co.uk/iplayer/added")
+    if not data:
+        return
+    has_episodes = False
+    for added_item in data['items']['elements']:
+        programme = added_item['programme']
+        ct_mnu = [('Remove',
+                   f'RunPlugin(plugin://plugin.video.iplayerwww?mode=302&episode_id={programme["id"]}&url=url)')]
+        if programme['count'] == 1:
+            CheckAutoplay(context_mnu=ct_mnu, **ParseEpisode(programme['initial_children'][0]))
+            has_episodes = True
+        else:
+            AddMenuEntry(mode=128, subtitles_url='', context_mnu=ct_mnu, **ParseProgramme(programme))
+    if has_episodes:
+        SetSortMethods(xbmcplugin.SORT_METHOD_DATE)
+    else:
+        SetSortMethods()
+
+
+def RemoveFavourite(programme_id):
+    """Remove an item from the 'Added' list.
+    Handler for the context menu option 'Remove' on list items in 'Added'.
+
+    """
+    from resources.lib.ipwww_common import headers as common_headers
+    requests.delete('https://user.ibl.api.bbc.co.uk/ibl/v1/user/adds/' + programme_id,
```

Comment:
I'd prefer to move this to ipwww_common, and keep ipwww_video without the dependency on requests.

Reply:
Yes, that's better. Is done.